src/components/DriveVideo/index.jsx

```diff
@@ -204,6 +204,7 @@ class DriveVideo extends Component {
     const { dispatch, isBufferingVideo, currentRoute } = this.props;
     if (!currentRoute) {
       dispatch(updateSegments());
+      // FIXME: currentRoute can't be true if it was just false?
       if (currentRoute && isBufferingVideo) {
```

Comment:
But removing `currentRoute` so this is executed again implicitly fixes this, though I'm not sure what the purpose of this is, it's extremely confusing.